cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DistributedPreviewManager.java

```diff
@@ -151,9 +151,15 @@ public void run() {
 
         Path runDir = Files.createTempDirectory(tmpDir, "preview");
         try {
+          CConfiguration cConfCopy = CConfiguration.copy(cConf);
           Path cConfPath = runDir.resolve("cConf.xml");
+          if (!cConf.getBoolean(Constants.Twill.Security.WORKER_MOUNT_SECRET)) {
+             // Unset the internal certificate path since certificate is stored cdap-security which
```

Comment:
Can we open a JIRA and track this as a TODO, as this will no longer work properly when SSL verification is enabled?

Reply:
Done